src/custom-element.js

```diff
@@ -594,67 +601,102 @@ function createBaseCustomElementClass(win) {
     }
 
     /**
-     * If the element has a media attribute, evaluates the value as a media
-     * query and based on the result adds or removes the class
-     * `i-amphtml-hidden-by-media-query`. The class adds display:none to the
-     * element which in turn prevents any of the resource loading to happen for
-     * the element.
-     *
-     * This method is called by Resources and shouldn't be called by anyone
-     * else.
-     *
+     * Part of CE API. Called when a subscribed attribute has changed. See
+     * `observedAttributes` for the set of attributes.
+     * @param {string} name
      * @final
-     * @package
      */
-    applySizesAndMediaQuery() {
-      assertNotTemplate(this);
+    attributeChangedCallback(name) {
```

Comment:
Ditto not implemented in CE polyfill.

Reply:
Let's dedupe to [this thread](https://github.com/ampproject/amphtml/pull/31948#discussion_r557003634).